connector/checkpoint/checkpoint_view.xml

```diff
@@ -48,6 +48,7 @@
                         delete="false" edit="false" version="7.0">
                     <field name="model_id"/>
                     <field name="record"/>
+                    <field name="message"/>
```

Comment:
well, without this, if you have a message only record you get a line with no meaning at all.
OTOH we could use  `name` and remove `model_id`, `record` and `message` at once.
Maybe we just need to add the description of the model in the name instead of just the display name.
What do you think?